### ISSUE TITLE
get yolo "gesture" worky

### ISSUE THREAD

**Issue by moz4r** [MyRobotLab/inmoov#180]
Seem it is related to a personal build, we need to use yolo opencv filter

**Comment by kwatters** [comment]
What do you want the gesture to do?  add the yolo filter?  what else?

**Comment by moz4r** [comment]
I think it is for "what do you see"

**Comment by kwatters** [comment]
you want to be able to respond with the current objects that are seen?  
yolo publishes that it recognizes on each frame.
I think you'd want to be able to ask the yolo filter what the last recognized objects are..  yes?

**Comment by moz4r** [comment]
you right, I think we have tools to play with it

**Comment by kwatters** [comment]
yeah. the info should be available in the "lastResult" object on the OpenCVFilterYolo  class..

**Comment by moz4r** [comment]
![image](https://user-images.githubusercontent.com/18124594/46438231-a3469d00-c75d-11e8-9bcb-7ea6d0d1742c.png)

**Comment by moz4r** [comment]
We are close.. classification publisher need to be converted to java land, because odd race conditions between python sleep / speech recognition …
